internal/schema/connection.go

```diff
@@ -145,7 +357,7 @@ func GetKibanaConnectionSchema() *schema.Schema {
 					RequiredWith: []string{"kibana.0.username"},
 				},
 				"endpoints": {
-					Description: "A list of endpoints where the terraform provider will point to, this must include the http(s) schema and port number.",
+					Description: "A comma-separated list of endpoints where the terraform provider will point to, this must include the http(s) schema and port number.",
```

Comment:
```suggestion
					Description: "A list of endpoints where the terraform provider will point to, this must include the http(s) schema and port number.",
```

It's not comma separated though, it's a TF list.